### ISSUE TITLE
Unable to resize Index and Filter by regions

### ISSUE THREAD

**Issue by BendixSpring** [TeamMentor/TM_4_0_Design#1017]
It would be nice to allow more real estate for looking at articles if we could resize or at least minimize the Index and Filter by regions by dragging the boundary to one side or clicking a shrink button.

**Comment by luchia** [comment]
For the time being, you can toggle the filters by clicking the filter icon -

![screen shot 2015-09-18 at 11 39 39 am](https://cloud.githubusercontent.com/assets/4547737/9950096/fbe524a2-5df9-11e5-80ea-af50e6c2e7e0.png)
![screen shot 2015-09-18 at 11 39 43 am](https://cloud.githubusercontent.com/assets/4547737/9950097/fbe92f20-5df9-11e5-99f2-fc401bb92ca5.png)

**Comment by romichg** [comment]
Filters already slide in and out. @luchia how hard would it be to add similar functionality for lefthand side? I will put this a P2 for now. We can discuss if we have time.

**Comment by luchia** [comment]
It wouldn't be hard, we'd just need Dinis to make the column toggle-able.

**Comment by luchia** [comment]
As per our discussions on our meeting, we think the index should be toggle-able on mobile, but not on desktop. This is ready for Dinis to wire up.

![screen shot 2015-09-26 at 6 46 56 pm](https://cloud.githubusercontent.com/assets/4547737/10116661/04cef09e-647f-11e5-8cef-8fa1499c57c7.png)
![screen shot 2015-09-26 at 6 47 00 pm](https://cloud.githubusercontent.com/assets/4547737/10116662/04cfc4ce-647f-11e5-84d2-179cb912f665.png)

**Comment by BendixSpring** [comment]
Not on the desktop?  The whole point is that if you have a gigantic monitor, being able to size things allows you to maximize your monitor.  Don’t most programmers have 2 or more big monitors?

--Danny

**Comment by romichg** [comment]
True, this is what is being done for now. The ability to toggle it on the desktop will come later :)
